WalletWasabi/WabiSabi/Client/CoordinatorClient.cs

```diff
@@ -61,5 +63,38 @@ public CoordinatorClient(WabiSabiClient wabiSabiClientAmount, WabiSabiClient wab
 
 			return inputRegistrationResponse.AliceId;
 		}
+
+		public async Task ConfirmConnectionAsync(Guid roundId, Guid aliceId, IEnumerable<Credential> credentialsToPresent, IEnumerable<Money> newAmount)
+		{
+			var (realAmountCredentialRequest, realAmountCredentialResponseValidation) = WabiSabiClientAmount.CreateRequest(
+				newAmount.Select(x => x.Satoshi),
+				credentialsToPresent);
+
+			var inputsCount = 1; // ?
```

Comment:
I think the only situation in which we should consider >1 is when registering multiple inputs with the same `scriptPubKey`, so this is correct

Reply:
Yes. The question is should this be received as parameter?

Reply:
i think it's simpler to just pass in the total input weight, this is known when the input(s) are registered and by connection confirmation it does not matter at all how many inputs there are, just the total remaining amount and the total remaining weight after the per-registration allocation